.github/workflows/sync.yml

```diff
@@ -0,0 +1,50 @@
+name: Sync DockerHub images with GHCR 🔁
+
+## This workflow is a temporal solution for having all our docker images synced to GHCR. However,
+## as soon as all the repo's CI is migrated to GitHub Actions, this can be removed.
+##
+## Project tracking the progress: https://github.com/orgs/jellyfin/projects/31
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Image sync 🔁
+    # Errors will probably be caused by excesses in API quota, so we can safely continue.
+    # Remaining workflows will be removed in the next scheduled run.
+    continue-on-error: true 
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - 'jellyfin/jellyfin'
+
+    steps:
+      - name: Clone repository
+      - uses: actions/checkout@v2.3.4
+
+      ## Logging in to DockerHub allows for more pulls without hitting DockerHub's quota (100 vs 200).
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1.10.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
does this really need to be a secret ? just a thought :wink:

Reply:
This secret is the one used in Vue repo, so I figure out it's easier for core to simply reveal it to this repo and that's it.

But yes, not sure why we had it in the first place in Vue.

Reply:
okay, that's sound then! I too prefer one single point of configuration, so if it ever changes it can be changed with ease.